src/test/java/com/exactpro/th2/mstore/TestErrorCollector.java

```diff
@@ -88,6 +88,21 @@ void afterEach() {
         verifyNoMoreInteractions(eventRouter);
     }
 
+    @ParameterizedTest
+    @CsvSource({
+            "60000,MICROSECONDS",
+            "60,SECONDS",
+            "1,MINUTES",
+    })
+    void testDrainTaskParameters(String period, String timeUnit) throws Exception {
```

Comment:
note: I think JUnit is smart enough to convert value to a long - have you tried using `long period` as a parameter instead of a string?

Reply:
Good point, some version ago JUnit couldn't do that